doc/guide/emacs.md

```diff
@@ -29,6 +29,8 @@ You can then make `gxi` your scheme program by setting `scheme-program-name`:
 
 And you can now run Gerbil with `M-x run-scheme`.
 
+N.B. `gerbil-mode.el` had a breaking change as of version 0.16. See [here](https://github.com/vyzo/gerbil/issues/510) for migrating.
```

Comment:
What we have to say is that "up to v0.16 the editing mode file was gerbil.el. Starting with `actual-version`, gerbil mode has been migrated to `gerbil-mode.el`; see [#510](...)"

Reply:
Note that gerbil.el was shipped with v0.16 so the comment as is is factually incorrect.